.github/workflows/test.yml

```diff
@@ -9,7 +9,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        ruby-version: [2.6, 2.7, 3.0, 3.1, 3.2]
+        ruby-version: [2.7, 3.0, 3.1, 3.2]
```

Comment:
`2.7` had the EOL on 31.03.2023, WDYT about dropping it as well?

Reply:
Done 👍